applications/StructuralMechanicsApplication/python_scripts/adaptative_remeshing_structural_mechanics_analysis.py

```diff
@@ -164,23 +164,39 @@ def _CreateProcesses(self, parameter_name, initialization_order):
         if parameter_name == "processes":
             processes_block_names = ["recursive_remeshing_process"]
             if len(list_of_processes) == 0: # Processes are given in the old format
-                KratosMultiphysics.Logger.PrintWarning("AdaptativeRemeshingStructuralMechanicsAnalysis", "Using the old way to create the processes, this will be removed!")
+                KM.Logger.PrintWarning("AdaptativeRemeshingStructuralMechanicsAnalysis", "Using the old way to create the processes, this will be removed!")
                 from process_factory import KratosProcessFactory
                 factory = KratosProcessFactory(self.model)
                 for process_name in processes_block_names:
-                    if (self.project_parameters.Has(process_name) is True):
+                    if self.project_parameters.Has(process_name):
                         list_of_processes += factory.ConstructListOfProcesses(self.project_parameters[process_name])
             else: # Processes are given in the new format
                 for process_name in processes_block_names:
-                    if (self.project_parameters.Has(process_name) is True):
-                        raise Exception("Mixing of process initialization is not alowed!")
+                    if self.project_parameters.Has(process_name):
+                        raise Exception("Mixing of process initialization is not allowed!")
         elif parameter_name == "output_processes":
             pass # Already added
         else:
             raise NameError("wrong parameter name")
 
         return list_of_processes
 
+    def _ReInitializeSolver(self):
+        """ This reinitializes after remesh """
+        self._GetSolver().Clear()
+        # WE INITIALIZE THE SOLVER
+        self._GetSolver().Initialize()
+        # WE RECOMPUTE THE PROCESSES AGAIN
+        ## Processes initialization
+        for process in self._list_of_processes:
```

Comment:
```suggestion
        for process in self._GetListOfProcesses():
```

Reply:
also in other places, `_list_of_processes` is a "private" variable

Reply:
I will check, think that we updated several times the analysis since I created this class